src/handlers/getBalancesTokens.ts

```diff
@@ -19,7 +19,7 @@ function formatBalance(balance: any): FormattedBalance {
     symbol: balance.symbol,
     decimals: balance.decimals,
     price: balance.price,
-    amount: balance.amount,
+    amount: formatUnits(balance.amount, balance.decimals),
```

Comment:
this line would break the app if it were merged. `amount` is the string with decimals, not the formatted one

Reply:
which request would break it? `amount` in `getPricedBalances` return type is `bigint`. Without this the `amount` ends up being `bigint`.

Reply:
https://github.com/llamafolio/llamafolio-api/pull/687/files#diff-07782dfad3574927d093ff623028d743a4574ffe650216177ae09187e006444cR75

you're returning this to the client, it's a handler

Reply:
there's a 

```ts
Object.defineProperties(BigInt.prototype, {
  toJSON: {
    value: function (this: bigint) {
      return this.toString()
    },
  },
})
```

defined that does that already, if you're not importing this file (but you are), then you can stringify it, but not format it. The difference is it returns `1.234567` to the client instead of `1234567000000000000`

Reply:
I see https://github.com/llamafolio/llamafolio-api/blob/feat/full-balance/src/db/pool.ts#L4

The handler doesn't use the `pg` client anymore in with the changes in this PR so that file is not imported.
Do you propose copying the `BigInt` modifier over to this file?

Reply:
either this or 

```ts
amount: balance.amount.toString(),
```

but not `formatUnits`

Reply:
did this 
```ts
amount: balance.amount.toString()
```